Knoema.Client/Client.cs

```diff
@@ -93,28 +94,31 @@ public Client(string host, string clientId, string clientSecret, bool ignoreCert
 
 		private HttpClient GetApiClient()
 		{
+			if (_client == null)
+			{
 #if NET45
-			var clientHandler = new WebRequestHandler();
+				var clientHandler = new WebRequestHandler();
 #else
-			var clientHandler = new HttpClientHandler();
+				var clientHandler = new HttpClientHandler();
 #endif
 
-			if (_ignoreCertErrors)
+				if (_ignoreCertErrors)
 #if NET45
-				clientHandler.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
+					clientHandler.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
 #else
-				clientHandler.ServerCertificateCustomValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
+					clientHandler.ServerCertificateCustomValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
 #endif
 
-			clientHandler.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
-			clientHandler.CookieContainer = _cookies;
+				clientHandler.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
+				clientHandler.CookieContainer = _cookies;
 
-			var client = new HttpClient(clientHandler) { Timeout = TimeSpan.FromMilliseconds(HttpTimeout) };
+				_client = new HttpClient(clientHandler) { Timeout = TimeSpan.FromMilliseconds(HttpTimeout) };
```

Comment:
Create the client in cotr(), now there are some problems in a multithreaded environment